### PR TITLE
Add WinForms sample with WebView2

### PR DESCRIPTION
**Pull request by hung-ngduc**
Similar to #94, [WebView2 Runtime](https://developer.microsoft.com/en-us/microsoft-edge/webview2/#download-section) needs to be installed on client machines.

### CODE REVIEW

File: WinFormsWebView2/WinFormsWebView2/Browser/ExtendedBrowser.cs [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Security.Permissions;
+using System.Windows.Forms;
+
+namespace WinFormsWebView2
+{ 
+    internal class ExtendedWebBrowser : WebBrowser
+    {
```
Comment (leastprivilege) [inline WinFormsWebView2/WinFormsWebView2/Browser/ExtendedBrowser.cs:13]: Why is this file needed?

Comment (hung-ngduc) [reply WinFormsWebView2/WinFormsWebView2/Browser/ExtendedBrowser.cs]: It shouldn't be there, I removed it.

**Comment by leastprivilege** [comment]
thanks!
